code/game/turfs/simulated/floor/plating/misc_plating.dm

```diff
@@ -118,6 +118,23 @@
 	desc = "You get the feeling that nobody's bothered to actually make this water functional..."
 	icon_state = "water"
 	baseturfs = /turf/open/floor/plating/beach/water
+	slowdown = 2
+	var/obj/effect/overlay/water/water
+
+/turf/open/floor/plating/beach/water/Initialize()
```

Comment:
could you add this to` /turf/open/floor/holofloor/beach/water` & `/turf/open/water` too?

Reply:
For the holotiles sure, dont think this would look good with our other water tiles

Reply:
holotiles plz